### PR TITLE
mumble: replace python and qt514 with python3 and qt5

### PR DESCRIPTION
**Pull request by TredwellGit**
###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/mumble/default.nix [hunk 1]
```diff
@@ -25,7 +25,7 @@ let
     patches = (source.patches or [])
       ++ [ ./fix-rnnoise-argument.patch ];
 
-    nativeBuildInputs = [ pkg-config python qt5.qmake ]
+    nativeBuildInputs = [ pkg-config python3 qt5.qmake ]
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/mumble/default.nix:28]: Shouldn't this use libsForQt5 instead of passing qmake in?

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110746` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
5 packages built:
</summary>
<ul>
<li>ioquake3</li>
<li>mumble</li>
<li>mumble_overlay</li>
<li>murmur</li>
<li>quake3demo</li>
</ul>
</details>

File: pkgs/applications/networking/mumble/default.nix (hunk 1 above)
Comment (TredwellGit) [reply pkgs/applications/networking/mumble/default.nix:28]: In all-packages.nix mumble and murmur use callPackages but there is no libsForQt5.callPackages.

Comment (SuperSandro2000) [reply pkgs/applications/networking/mumble/default.nix:28]: I mean ``libsForQt5.callPackage``.

Comment (TredwellGit) [reply pkgs/applications/networking/mumble/default.nix:28]: Is it safe to replace callPackages with libsForQt5.callPackage?

Comment (SuperSandro2000) [reply pkgs/applications/networking/mumble/default.nix:28]: I am not sure. It could double wrap maybe.

Comment (TredwellGit) [reply pkgs/applications/networking/mumble/default.nix:28]: This sounds like something that should be handled in a separate pull request.

**Pull request merged**